src/test/kotlin/no/nav/tilleggsstonader/sak/vilkår/stønadsperiode/StønadsperiodeServiceTest.kt

```diff
@@ -225,6 +228,67 @@ class StønadsperiodeServiceTest : IntegrationTest() {
         }
     }
 
+    @Nested
+    inner class GjenbrukStønadsperioder() {
+        val forrigeBehandlingId = UUID.randomUUID()
+        val nyBehandlingId = UUID.randomUUID()
+
+        @BeforeEach
+        fun setup() {
+            val fagsak = testoppsettService.lagreFagsak(fagsak())
+            testoppsettService.lagre(
+                behandling(
+                    fagsak = fagsak,
+                    id = forrigeBehandlingId,
+                    status = BehandlingStatus.FERDIGSTILT,
+                ),
+            )
+            testoppsettService.lagre(
+                behandling(
+                    fagsak = fagsak,
+                    id = nyBehandlingId,
+                    status = BehandlingStatus.UTREDES,
+                    forrigeBehandlingId = forrigeBehandlingId,
+                ),
+            )
+        }
```

Comment:
Lagde et forslag på å bytte ut denne delen og lage en funksjon for å opprette revurdering [her](https://github.com/navikt/tilleggsstonader-sak/commit/06d47e76269f495f0cf6d96139c640c74b1d5cd0). Den er ikke så tilpasningsdyktig, men det er jo ikke et behov for det i disse tilfellene. Det kan evt. utvides senere. 

Let me know om jeg burde bytte ut

Reply:
Ja, kjør på 👍